docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -98,6 +98,188 @@ Interventions
+prevalence among the population exposed to vitamin A fortification relative to 
+the population unexposed to vitamin A fortification. Therefore, we manually 
+calculated this value based on data reported in study tables and figures, 
+which required visual approximations of certain values. Notably, when this 
+supplementary meta-analysis was performed, the resulting relative risk was 
+calcualted as **0.43 (95% CI: 0.28 - 0.65)**. However, when limited to sugar 
+and oil vehicles for the vitamin A forticant, the relative risk was **0.36 
+(95% CI: 0.26 - 0.50)**. These two supplementary meta-analyses are represented 
+in the forest plots below. 
+
+.. image:: vitamin_a_meta.png
+
+.. image:: vitamin_a_meta_sugar_oil.png
+
+While the supplementary meta-analysis shown above contains more studies and 
+data than the Cochrane review, it relies on results that were not directly 
+reported in the individual studies (and in some cases visaully estimated 
+values). **Therefore, we will conservatively use the results from the Cochrane 
+review, with increased certainty in the results based on the confirmatory 
+results from the supplementary meta-analysis.**
+
+Notably, all of these studies included in the supplementary analysis were 
+conducted among children and study locations included Guatemala, South Africa, 
+Nicaragua, Indonesia, and the Phillipines, so it is **reasonable to assume 
+generalizability of these results to our model populations.**
+
+Regarding effect sizes in young age groups, Sandjaja et al. (2015) reported 
+that population vitamin A fortification improved serum retinol concentrations 
+among infants aged 6-11 months. Therefore, **we assumed that the effect size 
+from the Cochrane review applies to all age groups above six months of age.** 
+
+	Notably, the effect can occur either through the direct consumption of 
+	vitamin A fortified foods or through the consumption of breastmilk from 
+	mothers who consume vitamin A fortified foods (Sandjaja et al. 2015; WHO 
+	Guidelines).
+
+For individuals aged between 0 and six months, we made the following 
+assumptions:
+
+	1. Maternal consumption of vitamin A fortified foods has no effect on 
+	infant vitamin A deficiency birth prevalence. This assumption is supported 
+	by studies performed by Dror and Allen (2018).
+
+	2. There is a scale up in effect size from no effect (RR=1) at birth to 
+	the full effect size (RR=0.45) at six months of age. While there was no 
+	data identified for the effect of vitamin A *fortification* in children 
+	less than six months of age, we found that there were mixed results on the 
+	impact of maternal vitamin A *supplementation* (via breastmilk) on three 
+	month old infants (as discussed by Martins et al. 2010) and based our 
+	assumption on this finding.
+
+	3. The scale-up in effect size from birth to six months of age occurs in a 
+	linear fashion. We made this assumption in the absence of supporting data.
```

Comment:
Technically, I think we're assuming a linear scale-up of the _reciprocal_ of the relative risk stated above.  If that's correct, maybe we should change this to something like "The scale-up in the reciprocal of the effect size from birth to six months of age occurs in a linear fashion," just to be clear.

Reply:
Yeah, agreed. I will update this.